src/Microsoft.AspNetCore.OData/EnableQueryAttribute.cs

```diff
@@ -420,5 +407,29 @@ public virtual void ValidateQuery(HttpRequest request, ODataQueryOptions queryOp
             Contract.Assert(model != null);
             return model;
         }
+
+        /// <summary>
+        /// Holds request level query information.
+        /// </summary>
+        private class RequestQueryData
```

Comment:
>RequestQueryData [](start = 22, length = 16)

can we use the "ODataFeature" ?

Reply:
Their is no intent to allow customers to change this value, thus exposing it in ODataFeature might give a sense that callers can change it, however changing it will result into break in queries.